scout/server/app.py

```diff
@@ -40,11 +40,11 @@
     from chanjo_report.server.app import configure_template_filters
     from chanjo_report.server.blueprints import report_bp
     from chanjo_report.server.extensions import api as chanjo_api
-except ImportError:
+except ImportError as error:
     chanjo_api = None
     report_bp = None
     configure_template_filters = None
-    LOG.info("chanjo report not installed!")
+    LOG.warning("chanjo-report not properly installed! %s", error)
```

Comment:
raise an error with the right message instead? So we realise immediately that chanjo-report wasn't installed instead of going to case page that crashes and perhaps we have to open a new PR to fix it?

Reply:
Ah, but no, we still want it to run without chanjo installed... Thats kind of the reason there is an except at all?

Reply:
Ah, but good, lets rephrase the warning (or not) depending on the configuration!

Reply:
Which I can't because the app isn't up yet there. Oh well.

Reply:
I think if you have chanjo settings in the config file (the mysql uri) you want to use coverage. In the situation when chanjo-report has problems (is not installed) the the app shouldn't start at all. This way you'd know immediately that you have to fix it. This is what I mean

Reply:
I'm not sure I follow you again: are you saying you are able to get the the report with a broken install of chanjo? That should not happen!